graphql_compiler/tests/test_helpers.py

```diff
@@ -38,6 +38,8 @@ def compare_ir_blocks(test_case, expected_blocks, received_blocks):
     for i in six.moves.xrange(len(expected_blocks)):
         expected = expected_blocks[i]
         received = received_blocks[i]
+        if expected != received:
+            __import__('pdb').set_trace()
```

Comment:
You should probably remove this before we merge into `master`.

Reply:
Woops, fixed

Reply:
I don't think it is, actually -- the `if expected != received:` line is still there and it's causing a syntax error.

Reply:
My bad. Hopefully actually fixed now.